frontends/web/src/routes/account/info/buyCTA.tsx

```diff
@@ -0,0 +1,41 @@
+import { FunctionalComponent, h } from 'preact';
+import { route } from 'preact-router';
+import { Button } from '../../../components/forms';
+import { translate, TranslateProps } from '../../../decorators/translate';
+import { Balances } from '../summary/accountssummary';
+import * as styles from './buyCTA.module.css';
+
+interface BuyCTAProps {
+    code?: string;
+    unit?: string;
+}
+
+type Props = BuyCTAProps & TranslateProps;
+
+const BuyCTAComponent: FunctionalComponent<Props> = ({code, unit, t}) => {
+    const onCTA = () => route(code ? `/buy/info/${code}` : `/buy/info`);
+    return (
+    <div className={`${styles.main} columns-container`}>
+        <h3 className="column">{t('accountInfo.buyCTA.information.looksEmpty')}</h3>
+        <h3 className="column">{t('accountInfo.buyCTA.information.start')}</h3>
+        <div className="column">
+            <Button primary onClick={onCTA}>{unit ? t('accountInfo.buyCTA.buy', {unit}) : t('accountInfo.buyCTA.buyCrypto')}</Button>
+        </div>
+    </div>);
+};
+
+export const BuyCTA = translate<BuyCTAProps>()(BuyCTAComponent);
+
+export const AddBuyOnEmptyBalances: FunctionalComponent<{balances?: Balances}> = ({balances}) => {
+    if (balances === undefined) {
+        return null;
+    }
+    const balanceList = Object.entries(balances);
+    if (balanceList.some(entry => entry[1].available.amount !== '0')) {
+        return null;
+    }
+    if ((balanceList.length === 1) && balanceList[0][1].available.unit === 'BTC') {
```

Comment:
Hi I am testing, here when there is more that 1 bitcoin account, this is showing 'Buy Crypto' but ideally it would then show 'Buy Bitcoin'